js/index.js

```diff
@@ -73,3 +73,42 @@ if (
 ) {
   document.getElementById("messages").style.visibility = "hidden";
 }
+const githubRequest = new XMLHttpRequest();
+githubRequest.open("GET", "https://api.github.com/users/klove2016/repos");
+
+githubRequest.send();
+
+githubRequest.onload = function () {
+  const repos = JSON.parse(githubRequest.responseText);
+
+  const projectSection = document.getElementById("projects");
+  const projectList = projectSection.querySelector("ul");
+  for (let i = 0; i < repos.length; i++) {
+    eachRepo = repos[i];
+    project = document.createElement("li");
+    const repoStrName = eachRepo.name.toString();
+    const repoNoDashName = repoStrName.replaceAll("-", " ");
+    const repoRealName = repoNoDashName.split(" ");
+    for (let i = 0; i < repoRealName.length; i++) {
+      if (repoRealName[i].length > 3) {
+        repoRealName[i] =
+          repoRealName[i][0].toUpperCase() + repoRealName[i].substr(1) + " ";
+      } else {
+        repoRealName[i] = repoRealName[i] + " ";
+      }
+    }
```

Comment:
I think you're on the right track here again, but this can be simplified. I'm not sure what the significance of length 3 is for capitalization, but assuming you want to keep that part, you could use `map` here to simplify a bit. You can also remove the additions of the spaces, and use `.join(" ")` later to handle that for you:

```suggestion
    const repoDisplayName = repoRealName.map(wordInName => (
      wordInName.length > 3 ? wordInName[0].toUpperCase() + wordInName.substr(1) : wordInName
    )).join(' ');
```